math/igraph/Portfile

```diff
@@ -18,21 +18,26 @@ platforms           darwin
 depends_lib         port:gmp \
                     port:libxml2
 
-checksums           rmd160  4471b3b8110ddefe85e164920ff3bbc4e7e81f83 \
-                    sha256  718a471e7b8cbf02e3e8006153b7be6a22f85bb804283763a0016280e8a60e95 \
-                    size    3625308
+checksums           rmd160  e000e51d3fe0457b746541d84b7f32e5cf4e8bf6 \
+                    sha256  cc935826d3725a9d95f1b0cc46e3c08c554b29cdd6943f0286d965898120b3f1 \
+                    size    3636192
 
 test.run            yes
 test.target         check
 
-# igraph 0.8.1 embeds GLPK 4.45. Currently GLPK is at version 4.65.
+# igraph 0.8.3 embeds GLPK 4.45. Currently GLPK is at version 4.65.
 # Some igraph functions perform better with this new GLPK version.
 
 variant external_glpk description {Build with external GLPK} {
     configure.args-append   --with-external-glpk
     depends_lib-append      port:glpk
 }
 
+variant external_linalg description {Build with external BLAS, LAPACK, ARPACK} {
+	configure.args-append	--with-external-blas --with-external-lapack --with-external-arpack
+	depends_lib-append		port:lapack port:arpack
```

Comment:
The whitespace on these lines doesn't conform to the modeline. (Use spaces, not tabs.)

Reply:
Thanks for the review! It was an accident due to using an unfamiliar editor. It is fixed now. The only change in the force-push is what you requested, so the result of CI tests will be the same.